source/registry.d

```diff
@@ -0,0 +1,679 @@
+module registry;
+
+import std.exception : basicExceptionCtors;
+
+version(unittest) import aurorafw.unit.assertion;
+
+
+/**
+ * Trait which checks if a type is a valid entity type. An entity type must be
+ *     unsigned and integral. These are all the accepted types ***ubyte, ushort,
```

Comment:
nitpick, bad grammar:

`...are all the accepted types: ***ubyte, ushort, ...`

Reply:
done